---
title: "'Exit Do' ne peut apparaître qu’à l’intérieur d’une instruction 'Do'"
ms.date: 07/20/2015
f1_keywords:
- bc30089
- vbc30089
helpviewer_keywords:
- BC30089
ms.assetid: 0e1d0b35-e42b-4b90-b8a2-91fd6ef44f06
ms.openlocfilehash: c20de10946d13654042d75f08585ecd2bfab0f2a
ms.sourcegitcommit: bf5c5850654187705bc94cc40ebfb62fe346ab02
ms.translationtype: MT
ms.contentlocale: fr-FR
ms.lasthandoff: 09/23/2020
ms.locfileid: "91070960"
---
# <a name="exit-do-can-only-appear-inside-a-do-statement"></a>'Exit Do' ne peut apparaître qu’à l’intérieur d’une instruction 'Do'

Une instruction `Exit Do` se trouve en dehors d’une boucle `Do` . `Exit Do` n’est valide qu’entre une instruction `Do` et une instruction `Loop` correspondante.  
  
 **ID d’erreur :** BC30089  
  
## <a name="to-correct-this-error"></a>Pour corriger cette erreur  
  
1. Vérifiez qu’une instruction `Do` valide précède `Exit Do` et qu’une instruction `Loop` valide le suit.  
  
2. Vérifiez que les autres structures de contrôle dans la boucle `Do` sont terminées correctement.  
  
## <a name="see-also"></a>Voir aussi

- [Do...Loop (instruction)](../language-reference/statements/do-loop-statement.md)
